Docker build broken at tag 1.1.89

The docker build is no longer working for me if I try to use tag:1.1.89

```bash
   Compiling lazy_static v1.4.0
   Compiling unsigned-varint v0.5.0
   Compiling multihash v0.0.1 (/usr/src/etl/_build/default/lib/multihash)
   Compiling unreachable v1.0.0
error[E0658]: the `#[non_exhaustive]` attribute is an experimental feature
  --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/unsigned-varint-0.5.0/src/decode.rs:25:1
   |
25 | #[non_exhaustive]
   | ^^^^^^^^^^^^^^^^^
   |
   = note: for more information, see https://github.com/rust-lang/rust/issues/44109

error: aborting due to previous error

For more information about this error, try `rustc --explain E0658`.
error: could not compile `unsigned-varint`.
warning: build failed, waiting for other jobs to finish...
error: build failed
===> Hook for compile failed!
```bash

--- a/Dockerfile
+++ b/Dockerfile
@@ -3,13 +3,18 @@ FROM erlang:22.3.2-alpine as builder
 RUN apk add --no-cache --update \
     git tar build-base linux-headers autoconf automake libtool pkgconfig \
     dbus-dev bzip2 bison flex gmp-dev cmake lz4 libsodium-dev openssl-dev \
-    sed wget rust cargo
+    sed wget cargo
+
+# Install Rust toolchain
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 
 WORKDIR /usr/src/etl
 
 ENV CC=gcc CXX=g++ CFLAGS="-U__sun__" \
     ERLANG_ROCKSDB_OPTS="-DWITH_BUNDLE_SNAPPY=ON -DWITH_BUNDLE_LZ4=ON" \
-    ERL_COMPILER_OPTIONS="[deterministic]"
+    ERL_COMPILER_OPTIONS="[deterministic]" \
+    PATH="/root/.cargo/bin:$PATH" \
+    RUSTFLAGS="-C target-feature=-crt-static"
 
 # Add our code
 ADD . /usr/src/etl/

